templates/transcript.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
@peterwilsoncc The intention here was to follow a spec defined by Podcastindex.org. Intentionally barebones because there may be players that are able to read the HTML. Using the `post.php` template makes sense for humans reading this, should there be two versions? Original comment here: https://github.com/10up/simple-podcasting/issues/28#issuecomment-1410934588

Reply:
Thanks for the great suggestion. This has been added.